src/Sulu/Bundle/CoreBundle/DependencyInjection/SuluCoreExtension.php

```diff
@@ -111,6 +111,8 @@ public function load(array $configs, ContainerBuilder $container)
             $this->initFields($config['fields_defaults'], $container);
         }
 
+        $this->initListBuilder($container, $loader);
```

Comment:
not really we should extract list-builder in an own component and create a ListBuilderBundle for that